### PR TITLE
DOC: rework INSTALL.rst.txt

### PR DESCRIPTION
**Pull request by rgommers**

### CODE REVIEW

File: INSTALL.rst.txt [hunk 1]
```diff
@@ -27,16 +31,16 @@ PREREQUISITES
...
 SciPy requires the following software installed for your platform:
 
-1) Python__ 2.6.x or newer
+1) Python__ 2.7 or >= 3.4
 
 __ http://www.python.org
 
-2) NumPy__ 1.6.2 or newer (note: SciPy trunk at times requires latest NumPy
+2) NumPy__ 1.7.1 or newer (note: SciPy trunk at times requires latest NumPy
    trunk).
```
Comment (larsmans) [inline INSTALL.rst.txt]: "trunk" -> "master" (or something descriptive for VCS noobs?)

File: INSTALL.rst.txt [hunk 2]
```diff
@@ -192,25 +156,16 @@ To run the full test suite use
 
    >>> scipy.test('full')
 
-Please note that you must have version 0.10 or later of the 'nose' test
+Please note that you must have version 1.0 or later of the 'nose' test
 framework installed in order to run the tests.  More information about nose is
 available on the website__.
 
 __ http://somethingaboutorange.com/mrl/projects/nose/
```
Comment (larsmans) [inline INSTALL.rst.txt]: Dead link, should be https://nose.readthedocs.org/en/latest/

File: INSTALL.rst.txt (hunk 1 above)
Comment (rgommers) [inline INSTALL.rst.txt]: This is simply not true anymore, so I'll just change the sentence to ``NumPy >= 1.7.1``

File: INSTALL.rst.txt (hunk 2 above)
Comment (rgommers) [inline INSTALL.rst.txt]: thanks, fixed

**Comment by rgommers** [comment]
TravisCI tests ran once, but now force-pushed with ``[ci skip]``.

File: INSTALL.rst.txt [hunk 3]
```diff
@@ -135,50 +109,39 @@ Use the command::
...
-(e.g. in /usr/lib/python3.4/site-packages/scipy or
+If you have an older version of Scipy installed and didn't install that with
+pip, you need to manually uninstall it first (remove from e.g. in
+/usr/lib/python3.4/site-packages/scipy or
 $HOME/lib/python3.4/site-packages/scipy).  Then type::
 
   cd scipy
   git clean -xdf
-  python setup.py install
+  pip install . --user
```
Comment (ev-br) [inline INSTALL.rst.txt]: So you do recommend `pip install <dot>` over `python setup.py`? This does not break incremental rebuilds anymore?

Comment (ev-br) [inline INSTALL.rst.txt]: (Apologies if this was decided on and I just missed that) I thought I would recommend using a virtualenv.

Comment (rgommers) [inline INSTALL.rst.txt]: It does. I would not recommend to use ``pip`` for development. But this doc is about installing, not developing. So it's for most people a one-time operation (assuming things go well) - and then the benefit of reliable uninstall is more important imho.

Comment (rgommers) [inline INSTALL.rst.txt]: Hmm, ``pip`` 8.0 contains a bug where it doesn't show the build log even though you ask it to with ``--verbose``. So I'll revert this change.

Comment (rgommers) [inline INSTALL.rst.txt]: updated

File: INSTALL.rst.txt [hunk 4]
```diff
@@ -393,9 +251,3 @@ where INSTALLDIR is, for example, /usr/lib/python3.4/site-packages/.
      ldd /path/to/ext_module.so
 
    can be useful.
-
```
Comment (Eric89GXL) [inline INSTALL.rst.txt]: The sentence "where INSTALLDIR is, for example" should be indented to render properly. Can you fix as part of this PR?

**Comment by Eric89GXL** [comment]
Otherwise LGTM

File: INSTALL.rst.txt (hunk 4 above)
Comment (rgommers) [inline INSTALL.rst.txt]: good catch, fixed.

**Comment by rgommers** [comment]
anyone want to green-button this one?

**Comment by Eric89GXL** [comment]
Yeah, I'll start with an easy one :)

**Comment by Eric89GXL** [comment]
Thanks @rgommers

**Comment by rgommers** [comment]
:)

**Pull request merged**